src/common/actions.js

```diff
@@ -0,0 +1,11 @@
+import { fetchUserAccount as _fetchUserAccount, UserAccountApiService } from '@edx/frontend-auth';
```

Comment:
So this file is of interest.  These are some acrobatics to hide the existence of UserAccountApiService and apiClient from the profile module.  As you can see here, the default implementation of `fetchUserAccount` needs an instance of `UserAccountApiService` to be passed to it, which then in turn needs the configured `apiClient`.  By doing this and exposing a "wrapped" `fetchUserAccount`, we can have the high-level MFE boilerplate configure the service and the organisms in it are none-the-wiser about what it took to set it up.  This helps support the modularity of the profile.

Reply:
It'd be nice if this code didn't have to exist, though.